### PR TITLE
Initial Logic and System Architecture: Create tiles backend structure (service, entity, repository)

### PR DESCRIPTION
**Pull request by Patertuck**
New PR of the backend for tiles, as I rebased it for master. Apologies for the confusions.

### CODE REVIEW

**Comment by C0DECYCLE** [comment]
Will look at it today👍

File: src/main/java/ch/uzh/ifi/hase/soprafs24/service/TileService.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        tileRepository.save(toUpdateTile);
+        tileRepository.flush();
+    }
+
+    public List<Tile> getTilesInSession(String SessionId) {
+        List<Tile> TileInSession = tileRepository.findAllBySessionId(SessionId);
+        return TileInSession;
+    }
+}
```
Comment (C0DECYCLE) [inline src/main/java/ch/uzh/ifi/hase/soprafs24/service/TileService.java:52]: Beautiful!

File: src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java [hunk 2]
```diff
@@ -32,7 +38,7 @@ public List<PlayerDTO> getPlayersInSession(@RequestBody String sessionId) {
         for (Player player : playersInSession) {
             playerNameDTOs.add(DTOMapper.INSTANCE.convertEntityToPlayerDTO(player));
         }
-        return  playerNameDTOs;
+        return playerNameDTOs;
```
Comment (C0DECYCLE) [inline src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java:41]: nice catch!

**Review by C0DECYCLE — APPROVED** [review]
Love it! Looks good to me. Thanks!

**Comment by C0DECYCLE** [comment]
Thanks to the previous great reviews. @paul891112 @Twhining @Croxsy

**Comment by Patertuck** [comment]
> Love it! Looks good to me. Thanks!

Thanks for the positive notes, really boosts my moral 🥺❤️

**Pull request merged**